src/ureq_dns_resolver.rs

```diff
@@ -140,17 +36,11 @@ impl InternalAgent {
             )
         }));
 
-        // Create the CertificateVerifierNoHostname object
-        let cert_verifier = CertificateVerifierNoHostname::try_from((
-            &webpki_roots::TLS_SERVER_ROOTS,
-            allowed_dns_servers.iter().map(|s| s.to_string()).collect(),
-        ))
-        .unwrap();
-
         // See rustls documentation for more configuration options.
         let tls_config = rustls::ClientConfig::builder()
```

Comment:
Do you really need to specify a custom TLS config? Can't you use the default one?

Reply:
Probably, but I am looking into the rustls docs to see.